ingestion/tests/integration/orm_profiler/test_orm_profiler_e2e.py

```diff
@@ -212,6 +217,32 @@ def test_profiler_workflow(self):
 
         assert not table.tableProfilerConfig
         assert profile.profileSample == 75.0
+        assert profile.profileSampleType == ProfileSampleType.PERCENTAGE
+
+        workflow_config["processor"]["config"]["tableConfig"][0][
+            "profileSampleType"
+        ] = ProfileSampleType.ROWS
```

Comment:
Can you add a specific number of rows (let's 2 or 3) and check that we indeed only got 2 or 3 rows.